index.html

```diff
@@ -117,7 +118,7 @@ <h2 class="my-recent-work" id="my-recent-work">MY RECENT WORK</h2>
 
       <section class ="work-grid">
```

Comment:
This project requirement and the suggestion from the previous reviewer was not addressed.

As it is a project requirement and we are here to learn, now is the best time to practice our skills. We need to refactor our code for this milestone to render the work projects from a JavaScript array of objects.

- [ ] Kindly, update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all HTML in that section is created when the page loads. :100:

Reply:
@nicupop729 Sorry, I don't understand what is wrong. The projects informations are already being stored and shown with JavaScript

Reply:
Hi @Gopxfs,
Thanks for reaching back!

The cars with the projects for the moment are rendered by the HTML when what we have to do here, is to refactor and add them on the page using only JavaScript. As you render dynamically the pop-up, we have to render every work card using JavaScript. 
Please find here a screenshot from the project requirements:
![image](https://user-images.githubusercontent.com/83596135/167912701-1a919203-4b86-4d13-8ca2-01be4defbd51.png)

Is it a bit more clear now?

Reply:
@Gopxfs, it is a bit confusing because you are not working in pair-programming too.
As this is a project requirement, kindly, bring your partners and build these features together following the driver-navigator approach. Collaborating brings many advantages, one of them is that you have someone to debate with. And your partner could help the team to overcome the obstacles.

Reply:
@nicupop729 Thank you, I got it. I also have to make the project cards using JS. I would love to do pair programming but my partner told me he is sick and we couldn't work together, but we will do more pair programming this week.